packages/api/src/services/mail.js

```diff
@@ -0,0 +1,17 @@
+const sgMail = require('@sendgrid/mail');
+
+sgMail.setApiKey('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');
```

Comment:
should this be in the environment?

Reply:
No. 
I'll put this in Travis's env. variables.

Reply:
So it should not be a plain string here, right?

Reply:
Yes.